v2/components/schemas/Snap.yaml

```diff
@@ -0,0 +1,96 @@
+# SPDX-License-Identifier: GPL-3.0
+# SPDX-FileCopyrightText: Canonical Ltd
+
+type: object
+description: Detailed information about a single snap.
+properties:
+  id:
+    type: string
+    description: The unique identifier for the snap.
+  name:
+    type: string
+    description: The name of the snap.
+  base:
+    type: string
+    description: The base snap this snap is built on.
+    example: core20
+  channel:
+    type: string
+    description: The default channel for this snap.
+  common-ids:
+    type: array
+    items:
+      type: string
+    example: code.desktop
+  confinement:
+    type: string
+    enum: 
+      - strict
+      - classic
+  contact:
+    type: string
+    description: The primary contact URL or email for the developer.
+  description:
+    type: string
+    description: A detailed description of the snap.
+  developer:
+    type: string
+    description: The username of the snap developer.
+  devmode:
+    type: boolean
+    description: Whether the snap can be installed in devmode.
```

Comment:
This is opposite. The snap requires to be installed with devmode confinement and doesn't, yet, work with strict confinement due to sandbox limitations or developer unfamiliarity.